docs/release_notes/destinations_v2.js

```diff
@@ -100,53 +107,120 @@ AS (
 )`;
   }
   return (
-    <MigrationGenerator destination="snowflake" generateSql={generateSql}/>
+    <MigrationGenerator destination="snowflake" generateSql={generateSql} />
   );
-}
+};
 
-export const MigrationGenerator = ({destination, generateSql}) => {
-  const defaultMessage =
-`Enter your stream's name and namespace to see the SQL output.
+export const RedshiftMigrationGenerator = () => {
+  // See RedshiftSQLNameTransformer
+  function redshiftConvertStreamName(str) {
+    str = convertStreamName(str);
+    if (str.charAt(0).match(/[A-Za-z_]/)) {
+      return str;
+    } else {
+      return "_" + str;
+    }
+  }
+  function generateSql(og_namespace, new_namespace, name, raw_schema) {
+    let v2RawTableName =
+      '"' + concatenateRawTableName(new_namespace, name) + '"';
+    let v1namespace = redshiftConvertStreamName(og_namespace);
+    let v1name = redshiftConvertStreamName("_airbyte_raw_" + name);
+    return `CREATE SCHEMA IF NOT EXISTS "${raw_schema}";
+DROP TABLE IF EXISTS "${raw_schema}".${v2RawTableName};
+CREATE TABLE "${raw_schema}".${v2RawTableName} (
+  "_airbyte_raw_id" VARCHAR(36) NOT NULL PRIMARY KEY
+  , "_airbyte_extracted_at" TIMESTAMPTZ DEFAULT NOW()
+  , "_airbyte_loaded_at" TIMESTAMPTZ
+  , "_airbyte_data" SUPER
+);
+INSERT INTO "${raw_schema}".${v2RawTableName} (
```

Comment:
While there is a `CREATE TABLE AS (...)` does [exist](https://docs.aws.amazon.com/redshift/latest/dg/r_CREATE_TABLE_AS.html), you can't pre-define the columns in the new table - you can only CAST as part of the select statement in the query to load the data.  However, while that will produce the right types, you can't set default values, e.g. `TIMESTAMPTZ DEFAULT NOW()`